config/software/expat.rb

```diff
@@ -36,7 +36,18 @@
 
   update_config_guess(target: "conftools")
 
+  # AIX needs two fixes to compile the latest version.
```

Comment:
Thank you for documenting why we're doing this. It's hard years later to figure out why these things were done